corehq/pillows/mappings/case_mapping.py

```diff
@@ -104,10 +106,19 @@
 
 CASE_ES_ALIAS = "hqcases"
 
+CASE_INDEX_META = deepcopy(DEFAULT_META)
+
+if settings.SERVER_ENVIRONMENT == 'production':
+    CASE_INDEX_META.update({
+        'settings': {
+            'number_of_shards': 10,
+        },
+    })
```

Comment:
we could do that, i'm a little hesitant because then our ES settings would live in ansible, HQ, and commcare-hq-deploy. i was already confused initially about where to make this change